action.yml

```diff
@@ -23,6 +23,10 @@ inputs:
       If github-token is specified, this is the run that artifacts will be downloaded from.'
     required: false
     default: ${{ github.run_id }}
+  if-no-name-found:
```

Comment:
Thoughts on `if-not-found` instead?

That way this can cover both cases for when the name is specified, and when the name is not specified (multiple artifacts).

Reply:
Implemented